microservice/pom.xml

```diff
@@ -24,6 +24,7 @@
         <guava.version>31.0.1-jre</guava.version>
         <googleauth.version>1.1.1</googleauth.version>
         <jackson.version>2.12.5</jackson.version>
+        <mockito.version>2.23.0</mockito.version>
```

Comment:
Are you downgrading mockito ? Is there a reason for that ?

Reply:
Good catch... Somehow I messed this up while resolving a merge conflict. Will be gone with next commit.